src/dynamodb-mcp-server/awslabs/dynamodb_mcp_server/server.py

```diff
@@ -159,6 +167,30 @@ def get_dynamodb_client(region_name: str | None):
 resource_arn: str = Field(description='The Amazon Resource Name (ARN) of the DynamoDB resource')
 
 
+@app.tool()
+@handle_exceptions
+async def dynamodb_data_modeling() -> str:
+    """Retrieves the complete DynamoDB Data Modeling Expert prompt.
+
+    This tool returns a production-ready prompt to help user with data modeling on DynamoDB.
+    The prompt guides through requirements gathering, access pattern analysis, and production-ready
+    schema design. The prompt contains:
+
+    - Structured 2-phase workflow (requirements → final design)
+    - Enterprise design patterns: hot partition analysis, write sharding, sparse GSIs, and more
+    - Cost optimization strategies and RPS-based capacity planning
+    - Multi-table design philosophy with advanced denormalization patterns
+    - Integration guidance for OpenSearch, Lambda, and analytics
+
+    Usage: Simply call this tool to get the expert prompt.
+
+    Returns: Complete expert system prompt as text (no parameters required)
+    """
+    prompt_file = Path(__file__).parent / 'prompts' / 'dynamodb_architect.md'
```

Comment:
What part might https://github.com/modelcontextprotocol/python-sdk/tree/main?tab=readme-ov-file#prompts play in this if any?